### DISCUSSION TITLE
DataTable sorting icons

### DISCUSSION THREAD

**Discussion by StackedCrooked** [primefaces/community#2276] [PrimeIcons] [answered]
I was just wondering about the default sorting icon used by the DataTable component. Specifically about this icon: 
<img width="77" alt="image" src="https://github.com/primefaces/community/assets/83649/288f34c4-1bf1-4c36-8a30-95a9e9f4eee9">

The arrow points from smallest to largest, so I'd think means ascending.

However, as illustrated in this example, it apparently means descending:
![image](https://github.com/primefaces/community/assets/83649/42e8e5c9-2e2e-4eb1-b796-4a369d4367a5)

 Is it using the wrong icon? Or am I misinterpreting icon symbols?

**Comment by onursenture** [comment]
Hello. I understand you are asking about the PrimeFaces (JSF) datatable from your shared screenshot. I checked the icon usage in our other libraries and found no issues. Since the problem is related more to the library than PrimeIcons, opening an issue in the [PrimeFaces repository](https://github.com/primefaces/primefaces) would be more helpful in resolving the problem. Thanks.

**Reply by StackedCrooked** [reply]
> Hello. I understand you are asking about the PrimeFaces (JSF) datatable from your shared screenshot. I checked the icon usage in our other libraries and found no issues. Since the problem is related more to the library than PrimeIcons, opening an issue in the [PrimeFaces repository](https://github.com/primefaces/primefaces) would be more helpful in resolving the problem. Thanks.

Just to clarify: I'm using the [DataTable](https://primereact.org/datatable/) component from the PrimeReact library. The icon is called "pi-sort-amount-down-alt" in this list here: https://primereact.org/icons/ . 

Should I post an issue in the [PrimeReact repository](https://github.com/primefaces/primereact)?

**Reply by onursenture** [reply, marked as answer]
Yes.

**Reply by StackedCrooked** [reply]
Posted here https://github.com/primefaces/primereact/issues/6852
